internal/tools/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/signalfx/splunk-otel-collector/internal/tools
 
-go 1.21
+go 1.21.0
+
+toolchain go1.21.10
```

Comment:
Is the `toolchain` directive required here?

Reply:
It was added by the tool itself, I didn't try remove it.